hm8_3.py

```diff
@@ -0,0 +1,33 @@
+""" Task 3: validate """
+
+
+def validate(code):
+
+    """ Function return the existance of numeric! card """
+
+    while True:
+        if str(code).isdigit() and len(str(code)) == 16:
+            code = list(str(code))
+            length = len(code)
+            sum_odd = 0
+            sum_even = 0
+            digit = 0
+            for i in range(length):
+                digit = int(code[i])
+                if i % 2 != 0:
+                    sum_odd += digit
+                else:
+                    doubled = digit * 2
+                    if doubled > 9:
+                        sum_even += doubled - 9
+                    else:
+                        sum_even += doubled
+            return (sum_even + sum_odd) % 10 == 0
+        break
+    return None
```

Comment:
return False

Reply:
done 👍, also according to this comment add changes to hm8_2.py, please